pallets/moonbeam-lazy-migrations/src/tests.rs

```diff
@@ -263,3 +264,270 @@ fn test_clear_suicided_mixed_suicided_and_non_suicided() {
 		);
 	})
 }
+
+fn count_keys_and_data_without_code() -> (u64, u64) {
+	let mut keys: u64 = 0;
+	let mut data: u64 = 0;
+
+	let mut current_key: Option<Vec<u8>> = Some(Default::default());
+	while let Some(key) = current_key {
+		if key.as_slice() == sp_core::storage::well_known_keys::CODE {
+			current_key = sp_io::storage::next_key(&key);
+			continue;
+		}
+		// print!("Key: {} ", hexdisplay::ascii_format(&key));
```

Comment:
Is this on purpose ? I would expect that either we print something or we remove the comment

Reply:
It was for debugging and making sure everything is working correctly. I've removed it now